### PR TITLE
#mayo4sam

### PR DESCRIPTION
**Pull request by iandioch**

### CODE REVIEW

File: steely/plugins/mayo.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    '{} #mayoforsam',
+]
+
+def mayo(message):
+    message += ' '
+    message = message.lower()
+    message = message.replace('the', 'de') # !important
+    message = message.replace('s', 'sh')
+    message = message.replace('sh ', 's ')
+    message = message.replace('shh', 'sh')
```
Comment (devoxel) [inline steely/plugins/mayo.py]: Add "?"

Comment (devoxel) [reply steely/plugins/mayo.py]: The line below this #phonereview

File: steely/plugins/mayo.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    sentences = re.split('[.,!]', message)
+    out = ''
+    for s in sentences:
+        if random.random() < 0.4:
+            out += random.choice(FORMATS).format(s) + ' '
+        else:
+            out += s + ' '
```
Comment (devoxel) [inline steely/plugins/mayo.py]: This might be nicer with a list and ' '.join

Comment (iandioch) [reply steely/plugins/mayo.py]: yeah but does mayo deserve "nice"

File: steely/plugins/mayo.py (hunk 1 above)
Comment (iandioch) [reply steely/plugins/mayo.py]: ara shure #mayo4sam

File: steely/plugins/mayo.py (hunk 2 above)
Comment (devoxel) [reply steely/plugins/mayo.py]: that is a solid point

File: steely/plugins/mayo.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    '{} ara.',
+    'ara {}.',
+    '{} #mayo4sam',
+    '{} #mayoforsam',
+]
+
+def mayo(message):
+    message += ' '
+    message = message.lower()
+    message = message.replace('the', 'de') # !important
```
[the remark this answers is not in the document]
Comment (iandioch) [reply steely/plugins/mayo.py]: not when you're textin boyo

File: steely/plugins/mayo.py [hunk 4]
```diff
@@ -0,0 +1,44 @@
+'''gives the previous command m a y o n n a i s e'''
+
+import random
+import re
+import string
+
+__author__ = ('iandioch')
+COMMAND = '.mayo'
```
Comment (devoxel) [inline steely/plugins/mayo.py]: dot in command name 💯

Comment (itsdoddsy) [reply steely/plugins/mayo.py]: creds to me for spotting this thank you mr aaron for stealing the spotlight sir

**Comment by sentriz** [comment]
thats ✔ some good👌👌
can i merge now

**Comment by iandioch** [comment]
yes sir

**Pull request merged**